### ISSUE TITLE
Add table referencing other languages in the Spanish README-ES.md

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#291]
All of the other languages have a table at the top of it referencing different languages.

![](https://user-images.githubusercontent.com/2754821/52739544-022ae700-2f86-11e9-9af5-1ce28f254326.png)

But the Spanish README doesn't have this table at all. 

So this needs to be added to the top of the [`README-ES.md`](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/README-ES.md) file.

```
<table>
    <tr>
        <!-- Do not translate this table -->
        <td><a href="./README.md"> English </a></td>
        <td><a href="./README-CN.md"> 中文 </a></td>
        <td><a href="./README-RU.md"> русский </a></td>
        <td><a href="./README-RO.md"> Românesc </a></td>
        <td><a href="./README-IT.md"> Italiano </a></td>
         <td><a href="./README-ES.md"> Español </a></td>
        <td><a href="./README-pt-BR.md"> Português (BR) </a></td>
    </tr>
</table>

```

**Comment by marieram** [comment]
@erictleung Can I work on this one?

**Comment by erictleung** [comment]
@marieram sure, go for it! I noticed this when making the last issue you helped out with. 

Just a tip for making pull requests on GitHub, you can add `Closes #291` in your pull request body message to reference this specific issue and also automatically close it when it is merged. You can read more [about it here](https://help.github.com/articles/closing-issues-using-keywords/). Good luck!
